lib-js/reporter.ts

```diff
@@ -0,0 +1,85 @@
+import {Writable} from 'stream';
```

Comment:
For files that export one primary class, the filename should match that classname (so `lint-reporter.ts`).

Reply:
😅